### ISSUE TITLE
[gamebanana] error installing 61281

### ISSUE THREAD

**Issue by Caliel666** [Brod8362/d4m#20]
it's a weird one, it's trying to install on the "maps" folder, even creating one

looking at it's files, other then possibly missing a mod_pv_db.txt file, it seems normal

![image](https://user-images.githubusercontent.com/38433434/181640557-200f4b73-f442-4ac4-a807-6d015d4615db.png)

```shell
Installing Dramaturgy -Eve cover- by MandM (61281)
Failed to install Dramaturgy -Eve cover- by MandM ([Errno 2] No such file or directory: '/mnt/games/SteamLibrary/steamapps/common/Hatsune Miku Project DIVA Mega Mix Plus/mods/maps/config.toml')
Traceback (most recent call last):
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/divamod.py", line 15, in diva_mod_create
    return DivaMod(path)
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/divamod.py", line 100, in __init__
    super().__init__(path)
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/divamod.py", line 23, in __init__
    with open(os.path.join(path, "config.toml"), "r", encoding="UTF-8") as mod_conf_fd:
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/games/SteamLibrary/steamapps/common/Hatsune Miku Project DIVA Mega Mix Plus/mods/maps/config.toml'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/tui.py", line 66, in menu_install
    mod_manager.install_mod(mod[0], origin=mod[2])
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/manage.py", line 106, in install_mod
    new_mod = diva_mod_create(mod_folder_name)
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/divamod.py", line 17, in diva_mod_create
    return DivaSimpleMod(path)
  File "/home/demuriel/.local/lib/python3.10/site-packages/d4m/divamod.py", line 23, in __init__
    with open(os.path.join(path, "config.toml"), "r", encoding="UTF-8") as mod_conf_fd:
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/games/SteamLibrary/steamapps/common/Hatsune Miku Project DIVA Mega Mix Plus/mods/maps/config.toml

**Comment by Brod8362** [comment]
This is... interesting...
ID 61281 is a Garry's mod TTT map. https://gamebanana.com/mods/download/61281
I need to figure out where it's getting that ID from, or more specifically *why* the API is returning it.

**Comment by Brod8362** [comment]
Ah, I see. That mod is under the `sounds` category: https://gamebanana.com/sounds/61281
So it's fetching a sound ID and treating it as a mod id.

**Comment by Brod8362** [comment]
Fixed in unstable, will be fixed in 0.3.3.

**Comment by Brod8362** [comment]
Fixed in v0.3.3.
